docker-compose.yml

```diff
@@ -104,6 +104,12 @@ services:
       BACKEND_PORT: ${BACKEND_PORT}
       JWT_SECRET: ${JWT_SECRET}
       WEBDOMAIN: "http://localhost:${FRONTEND_PORT}"
+      RABBITMQ_DEFAULT_USER: ${RABBITMQ_DEFAULT_USER}
+      RABBITMQ_DEFAULT_PASS: ${RABBITMQ_DEFAULT_PASS}
```

Comment:
Huh, I only now see I named these not very nicely. If you have a moment, can you remove the `_DEFAULT_` from the names? If not that OK too.

Reply:
Please remove DEFAULT from all the services in docker compose @wayangalihpratama. Update the .env.template, and also update the readme